cmd/tracee-ebpf: add capabilities dropping bypasses (#2008)
aquasecurity/tracee: docs/deep-dive/dropping-capabilities.md

without the user agreement**.

To **allow tracee to run with high capabilities** and prevent errors, the
`--allow-high-capabilities` flag can be used in tracee-rules, or
`--caps allow-failed-drop` in tracee-ebpf. For docker users, to allow
**tracee-ebpf** high capabilities the environment variable
`ALLOW_HIGH_CAPABILITIES=1` should be used.

## Missing Capabilities Errors

New features and refactoring might result missing capabilities for **tracee-ebpf**.

> This might cause wide variety of errors.

Reviewer: ```suggestion
This may cause a wide variety of errors.
```

> The Tracee team try to solve bugs as quickly as possible, but it might

Reviewer: ```suggestion
Our team tries to solve bugs as quickly as possible, but it might
```

still take some time to solve some bugs. Moreover, after a bug is fixed
it might take some time until a new version of Tracee is released.

> To solve missing capabilities errors locally, users can add capabilities

Reviewer: ```suggestion
To fix specific missing capabilities errors locally, users can add capabilities
```

using the `--caps add` flag to tracee-ebpf. For docker users, the environment variable
`CAPS_TO_PRESERVE=<list_of_capabilities>` should be used.

> In addition, the `--caps cancel-drop` flag could be used to cancel capabilities

Reviewer: ```suggestion
In addition, the `--caps cancel-drop` flag can be used to cancel capability
dropping in tracee-ebpf. For docker users, the environment variable
```
